hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -2508,6 +2509,29 @@ void testWriteMultipleCommitWithLegacyAndRowDataWrite(String indexType, boolean
     assertRowsEquals(rows, TestData.DATA_SET_SOURCE_INSERT);
   }
 
+  @ParameterizedTest
+  @MethodSource("indexAndLogBlockType")
+  void testRowDataWriteModeWithDifferentLogFormat(String index, String logBlockType) throws Exception {
```

Comment:
there is no need to test the default log format I think.

Reply:
Make sense, will remove test option for AVRO block.